apps/proxy/src/app/create-project/create-project.component.scss

```
@@ -0,0 +1,9 @@
+.container {
+    height: 100vh;
+}
+.link {
+    color: var(--color-link-color);
+}
+.forwardUrl {
```

```suggestion
.forward-url {
```change class name to